modules/client/src/main/java/org/apache/ignite/internal/client/ReliableChannel.java

```diff
@@ -143,21 +142,18 @@ public final class ReliableChannel implements AutoCloseable {
 
     /** {@inheritDoc} */
     @Override
-    public synchronized void close() throws Exception {
+    public synchronized void close() {
         closed = true;
 
         List<ClientChannelHolder> holders = channels;
 
-        IgniteUtils.closeAllManually(
-                () -> {
-                    if (holders != null) {
-                        for (ClientChannelHolder hld : holders) {
-                            hld.close();
-                        }
-                    }
-                },
-                connMgr::stop,
-                () -> shutdownAndAwaitTermination(streamerFlushExecutor, 10, TimeUnit.SECONDS));
+        if (holders != null) {
+            for (ClientChannelHolder hld : holders) {
+                hld.close();
+            }
+        }
+        connMgr.stop();
+        shutdownAndAwaitTermination(streamerFlushExecutor, 10, TimeUnit.SECONDS);
```

Comment:
This is not the same as the old `closeAllManually` call - we should wrap every `close()` in a try-catch so that everything is closed even if there was an exception.

We should also do this for `holders`.